docs/Developers/Development_Setup.rst

```diff
@@ -99,8 +110,8 @@ like to modify using
     (coala-venv)$ pip3 install -e <path/to/clone>
 
 You will then be able to edit the repository and have the changes take effect
-in your virtualenv immediately. You will also be able to use pip to manage
-your installation of the package should you need to install from a different
+in your virtualenv immediately. You will also be able to use pip3 to manage
+your installation of the package, you need to install from a different
```

Comment:
This is not a grammatical error. That was a perfectly valid sentence (I even told you so in the chat). Why did you change it?